src/lib/constants.ts

```diff
@@ -116,5 +118,25 @@ export const stakingContracts = [
       }
     ],
     enabled: FEATURE_FLAGS.ethFoxStakingV2
+  },
+  {
+    name: 'ETH-FOX V3',
+    owner: 'ShapeShift',
+    contractAddress: '0xc54B9F82C1c54E9D4d274d633c7523f2299c42A0',
```

Comment:
We'll need to use the actual contract address for V3 here - if it's not yet available, leave a `TODO` next to this line, e.g. `contractAddress: '0xc54B9F82C1c54E9D4d274d633c7523f2299c42A0', // TODO - Placeholder, update to actual V3 address when available`.

This will help us stay aware of the fact that this is not yet expected to be the correct address.

Reply:
fixed